index.html

```diff
@@ -1,21 +1,123 @@
-<!DOCTYPE html>
-<html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>TV Show Project | My Name (My GitHub username)</title>
-    <link href="style.css" rel="stylesheet" />
-    <link rel="icon" href="/favicon.ico" type="image/x-icon" />
-  </head>
-
-  <body>
-    <div id="root">
-    </div>
-
-    <!-- Loads a provided function called getAllEpisodes() which returns all episodes -->
-    <script src="episodes.js"></script>
-
-    <!-- Loads YOUR javascript code -->
-    <script src="script.js"></script>
-  </body>
-</html>
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Buddy Project | Charity website</title>
+  <link href="style.css" rel="stylesheet" />
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link href="https://fonts.googleapis.com/css2?family=Judson&display=swap" rel="stylesheet" />
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@600&family=Judson&display=swap" rel="stylesheet" />
+</head>
+
+<body>
+  <header>
+    <div class="logo-div"> <img id="logo" style="cursor: pointer;" src="/Vector/logo.png"></img> </div>
+    
+    <div class="label-select"> 
+    <label>Where do you like to donate?</label>
+    <select id ="bar">
+      <option></option>
+      <option value="techOpt">Technology</option>
+      <option value= "aniOpt">Animal</option>
+      <option value="recOpt">Recycling</option>
+    </select>

```

Comment:
I like your use of a select element to view different pages. Remember to use a `for` attribute in the label that matches the `id` of the select tag - this links together the two elements for accessibility. Also a `name` attribute is used on the select element when it's submitted to the server - you can read more about these points here:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select